pkg/inngest/inngest/_internal/client_lib/client.py

```diff
@@ -423,6 +431,19 @@ def _get_steps_sync(
 
         return data
 
+    def _seed(self) -> str:
```

Comment:
Nit: this could be a pure function since it doesn't need `self`. It isn't a big deal to make it a class method, but it might be 1% easier to understand that it's independent of the client object